<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEditMain">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>753</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>메뉴</string>
    </property>
    <widget class="QMenu" name="menu_1">
     <property name="title">
      <string>테스트1</string>
     </property>
     <addaction name="action_1_1"/>
     <addaction name="separator"/>
    </widget>
    <addaction name="menu_1"/>
    <addaction name="actionMy"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_1_1">
   <property name="text">
    <string>테스트1-1</string>
   </property>
  </action>
  <action name="actionMy">
   <property name="text">
    <string>테스트2</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
